conf.d/http_check.yaml.example

```diff
@@ -7,6 +7,15 @@ instances:
     url: http://some.url.example.com
     timeout: 1
 
+    # The default HTTP method is GET but the (optional) method parameter allows you to change the
+    # HTTP method to POST.
+    # If using the POST method, you can then specify the data to send in the body of the request
+    # with the data parameter.
+    #
+    # method: post
```

Comment:
Since the default method is `get`, this should probably read `method: get` in case someone accidentally comments out.